person_capture/gui_app.py

```
@@ -5389,14 +5504,351 @@ def _resolve_ffmpeg_bin(self) -> Optional[str]:
+                z_desat = max(0.0, z_desat - 0.10)
+            elif gamut == "relative":
+                z_desat = max(0.0, z_desat - 0.05)
+            zf = (
+                f"{crop},{src},"
+                "zscale=primaries=bt2020:transfer=smpte2084:matrix=bt2020nc,"
+                "zscale=transfer=linear:npl=1000,format=gbrpf32le,"
+                f"tonemap=tonemap={z_algo}:param={z_param:.6g}:desat={z_desat:.6g}:peak={z_peak:.6g},"
+                f"zscale=transfer=bt709:primaries=bt709:matrix=bt709:dither={z_dither},"
+                "zscale=rangein=pc:range=pc,format=bgr24"
+            )
+            cmds.append(base + ["-vf", zf] + enc)
```

Ruff (0.15.11)

[warning] Consider `[*base, "-vf", zf, *enc]` instead of concatenation

Replace with `[*base, "-vf", zf, *enc]`

(RUF005)